src/components/CRUD.js

```diff
@@ -280,6 +297,16 @@ class CRUD extends React.Component {
                   actions.insertImage,
                   actions.insertVideo,
                 ]}
+                iconMap={{
+                  [actions.setBold]: ({ tintColor }) => (<>{setBoldSvg({ width: 25, height: 25, fill: tintColor })}</>),
+                  [actions.setItalic]: ({ tintColor }) => (<>{setItalicSvg({ width: 25, height: 25, fill: tintColor })}</>),
```

Comment:
why is this needed? I think the icons have an "on" state when clicked even without this mapping

Reply:
I have added mapping to use custom icons as per design update.